config/initializers/constants/environment.rb

```diff
@@ -2,4 +2,6 @@ module ENVIRONMENT
   STAGING = 'staging'.freeze
   INTEGRATION = 'integration'.freeze
   PRODUCTION = 'production'.freeze
+  TEST = 'test'.freeze
+  DEVELOPMENT = 'development'.freeze
```

Comment:
do we still need the constants, given the values are now being passed from the env?